src/main/java/org/fundacionjala/coding/dennis/CompDNA.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.HashMap;
+import java.util.Map;
+/**
+ *
+ */
+public final class CompDNA {
+
+    /**
+     *
+     */
+    private CompDNA() {
+    }
+
+    /**
+     *
+     * @param st Cadena
+     * @return Cadena
+     */
+    public static String cDna(final String st) {
+        Map<String, String> comp = new HashMap<String, String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)